### PR TITLE
[fix] Fix build

### PR DESCRIPTION
**Pull request by baseballyama**
Follow up: https://github.com/sveltejs/kit/pull/4235

The latest release was failed due to some build configuration.
So I tried to fix it.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [x] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -380,10 +380,11 @@ importers:
   sites/kit.svelte.dev:
     specifiers:
       '@sveltejs/adapter-auto': workspace:*
-      '@sveltejs/adapter-static': workspace:^1.0.0-next.28
+      '@sveltejs/adapter-static': workspace:^1.0.0-next.29
```
Comment (Conduitry) [inline pnpm-lock.yaml:383]: The static adapter should presumably be completely removed from the kt.svelte.dev site.

Comment (baseballyama) [reply pnpm-lock.yaml]: Maybe it's just my local, but after removing` @sveltejs/adapter-static`, `pnpm build` is failed.

**Comment by PH4NTOMiki** [comment]
Can you try to merge it @Conduitry

**Comment by benmccann** [comment]
Is every project going to need `devalue`? I don't think that would be a good situation

I'm guessing the need for it might be related to this: https://github.com/sveltejs/kit/pull/3942#issuecomment-1041622382

**Comment by Conduitry** [comment]
@benmccann `devalue` is only needed, I believe, because of the changes in https://github.com/sveltejs/kit/pull/4221/commits/ab4ef9cf5130bb30169e60444331352365f68ba6 - I wouldn't say no to reverting those as well, because it's clearly not the proper solution. (I'm not even sure whether Rich intended that to be part of the final PR.)

**Comment by Rich-Harris** [comment]
It would be nice to get rid of `devalue` if we can, so this problem doesn't pop up. The rationale behind ab4ef9cf5130bb30169e60444331352365f68ba6 was to make it easier to debug Kit while working on kit.svelte.dev, and I do think there's some value in preserving that — definitely wouldn't be upset if we decided against it but for now I reckon I'll merge this so that things are at least working again

**Review by Rich-Harris — APPROVED** [review]

**Comment by PH4NTOMiki** [comment]
@Rich-Harris thanks for merging that

**Comment by PH4NTOMiki** [comment]
@Rich-Harris I think you need to rerun that action that failed previously.
not sure thought

**Comment by baseballyama** [comment]
Maybe should we swap (rebase) latest 2 commits for releasing?

**Comment by bitdom8** [comment]
Latest build doesn't run 

    pnpm run build

ReferenceError: require is not defined in ES module scope, you can use import instead
This file is being treated as an ES module because it has a '.js' file extension and 'C:\Users\Administrator\Documents\client\package.json' contains "type": "module". To treat 
it as a CommonJS script, rename it to use the '.cjs' file extension.

**Pull request merged**